src/server/ua_services_attribute.c

```diff
@@ -1402,6 +1415,51 @@ __UA_Server_write(UA_Server *server, const UA_NodeId *nodeId,
     return UA_Server_write(server, &wvalue);
 }
 
+#ifdef UA_ENABLE_HISTORIZING
+
+#include "ua_plugin_history_data_service.h"
+
+void
+Service_HistoryRead(UA_Server *server,
+                    UA_Session *session,
+                    const UA_HistoryReadRequest *request,
+                    UA_HistoryReadResponse *response) {
+    UA_HistoryReadResponse_init(response);
```

Comment:
This overrides the `requestHandle` and `timestamp` of the response, which areset earlier in `processMSG()`. This does not work for async clients, like Milo. Furthermore, it's already initialized in `processMSG()` via `UA_init()`.

Reply:
Removed